test/lib/test_output.ml

```diff
@@ -0,0 +1,50 @@
+open Mdx.Output
+
+let test_of_string =
+  let make_test ~line ~expected =
+    let test_name = Printf.sprintf "of_string: %S" line in
+    let test_fun () =
+      let actual = of_string line in
+      Alcotest.(check (list Testable.sub)) test_name expected actual
+    in
+    (test_name, `Quick, test_fun)
+  in
+  [
+    make_test ~line:"" ~expected:[ S "" ];
+    make_test ~line:"foo" ~expected:[ S "foo" ];
+    make_test ~line:".." ~expected:[ S ".." ];
+    make_test ~line:"..." ~expected:[ S ""; Sub_ellipsis; S "" ];
```

Comment:
Any particular reason why this is not simply parsing to `Sub_ellipsis`? I know it's a bit of a corner case since that will actually be recognized as an `Ellipsis` and not an `Output` but I'm more generally suprised by the leading `S ""` whenever the line starts with an ellipsis.

Reply:
I added more cases when parsing the string, it's fixed now